spec.yaml

```diff
@@ -165,6 +165,59 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v4/info/app-catalog/:
```

Comment:
although, no, sorry, taking that back, the catalogs are in the backend, this might be only based on the app-release CRs which are there no matter which catalog their from, which reminds me, we might need a way to track back which catalog an app belongs to, right?

Reply:
How about just `/v4/info/apps/` instead of `/v4/info/app-catalog/`?